backend/fms_core/resources/sample.py

```diff
@@ -175,7 +175,6 @@ def import_obj(self, obj, data, dry_run):
         except Exception as e:
             individual_created = False
             individual = None
-            errors["individual"] = ValidationError(e.messages.pop(), code="invalid")
```

Comment:
If it's ok to not have an individual associated to a sample, an exception should not be thrown at all for that. So I would put a condition before attempting to get/create an individual

Reply:
I would test if there is an individual id i would only try to create or get an individual if there is one (i dislike using exception to deal with known rules). If there is no individual ID but there is other individual column populated(cohort, pedigree, mother, father, taxon, sex), i would raise an error saying the user need to provide an individual ID if said field(cohort, pedigree, mother, father, taxon, sex) is provided to be saved.